automation/scripts/bundle_demos.py

```diff
@@ -51,6 +66,22 @@
 # Copying update_demos.sh from mlrun
 shutil.copy2("automation/scripts/update_demos.sh", "demos/update_demos.sh")
 
+for demo, files in demos_files.items():
+    print(f"Downloading {len(files)} files for demo {demo}...")
+    for url in files:
+        file_name = os.path.join(temp_dir, "data", demo, url.split("/")[-1])
+        os.makedirs(os.path.join(temp_dir, "data", demo), exist_ok=True)
+        print(f"Downloading {url}...")
+        # Download the file and save it
+        try:
+            response = requests.get(url)
+            with open(file_name, "wb") as f:
+                f.write(response.content)
+        except Exception as e:
+            print(f"could not download {url}")
+            print(e)
```

Comment:
You can remove the print. The message is already printed by `traceback.print_exc()`.
```suggestion
```